app/models/request.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Request < ApplicationRecord
+  belongs_to :requester, class_name: 'Organisation', foreign_key: 'requester_id'
+  belongs_to :requestee, class_name: 'User', foreign_key: 'requestee_id'
```

Comment:
Is requestee a bad name? `request.requestee` feels a little silly... `request.recipient`?